src/components/select-project/selectProject.html

```diff
@@ -1,9 +1,23 @@
-<ng-form name="$ctrl.forms.selectProjectForm">
+<span ng-if="$ctrl.noProjectsCantCreate" class="no-projects-cant-create">
+  <pf-empty-state config="$ctrl.noProjectsConfig"></pf-empty-state>
+  <div ng-if="!$ctrl.noProjectsCantCreateMessage">
+    A cluster admin can create a project for you by running the command:
+    <div class="code-block">
+      <code>oc adm <span class="command-arg">new-project</span> &lt;projectname&gt; <span class="command-arg">--admin={{$ctrl.user.metadata.name || '&lt;YourUsername&gt;'}}</span></code>
```

Comment:
Is `$ctrl.user.metadata.name` ever set?

Reply:
Nice catch!  Thanks, fixed:
![image](https://user-images.githubusercontent.com/12733153/31728818-5a36fa72-b3fb-11e7-8067-6ee7a677ab90.png)